README.md

```diff
@@ -0,0 +1,65 @@
+# Future Days Farm
+A website for the [Future Days Farm](InsertLinkToWebsite)
+
+## Prerequisites
+* Download [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)
+* Download [npm and Node.js](https://nodejs.org/en/)
+
+### macOS
+* `$ npm install n -g`
+* `$ sudo n 9.3.0`
+* `$ git clone https://github.com/FutureDaysSoftware/FarmWebsite.git`
+* [Connect to GitHub with SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)
+* `$ cd ~/FarmWebsite/`
+* `$ npm install`
+* `$ rm ~/FarmWebsite/client/js/views/Toast.js`
+* `$ rm ~/FarmWebsite/client/js/views/templates/Toast.js`
+* `$ rm ~/FarmWebsite/stylus/include/Toast.styl`
+* `$ ln -s ~/FarmWebsite/node_modules/toast/views/Toast.js ~/FarmWebsite/client/js/views/Toast.js`
+* `$ ln -s ~/FarmWebsite/node_modules/toast/views/templates/Toast.js ~/FarmWebsite/client/js/views/templates/Toast.js`
+* `$ ln -s ~/FarmWebsite/node_modules/toast/stylus/Toast.styl ~/FarmWebsite/stylus/include/Toast.styl`
+
+### Windows
+
+### Linux
+
+## Setting Up Environment
+* Create a file called `.env` in the `~/FarmWebsite/` directory
+* Add these environment variables into the `.env` file:
+```
+COOKIE=""
```

Comment:
UPDATE: ScottAP108 -- has introduced a config based on node environment `process.env.NODE_ENV` ( https://nodejs.org/api/process.html#process_process_env )

No need to change this at the moment, just an FYI.